Lambda function fails to read from bucket

Lambda function reported the following error: 
`[ERROR] ClientError: An error occurred (403) when calling the HeadObject operation`

This is due to the permissions being set up in the template as `arn:aws:s3:::twimbot-bucket` rather than `arn:aws:s3:::twimbot-bucket/*`.

--- a/template.yaml
+++ b/template.yaml
@@ -68,7 +68,10 @@ Resources:
                         Action:
                             - s3:GetObject
                             - s3:ListBucket
-                        Resource: !GetAtt TwimbotBucket.Arn
+                        Resource: !Join
+                            - ""
+                            - - !GetAtt TwimbotBucket.Arn
+                              - "/*"
                 - Statement:
                       - Sid: DynamoDBReadWriteTable
                         Effect: Allow

